aws/table_aws_iam_service_specific_credential.go

```diff
@@ -94,17 +94,19 @@ func listAwsIamUserServiceSpecificCredentials(ctx context.Context, d *plugin.Que
 
 	userData, _ := svc.ListServiceSpecificCredentials(ctx, params)
 	if err != nil {
-		plugin.Logger(ctx).Error("aws_iam_user.listAwsIamUserServiceSpecificCredentials", "api_error", err)
+		plugin.Logger(ctx).Error("aws_iam_service_specific_credential.listAwsIamUserServiceSpecificCredentials", "api_error", err)
 		return nil, err
 	}
 
-	if userData.ServiceSpecificCredentials != nil {
-		for _, cred := range userData.ServiceSpecificCredentials {
-			d.StreamListItem(ctx, cred)
+	if userData != nil {
```

Comment:
```suggestion
	if userData != nil && userData. ServiceSpecificCredentials != nil {
```

Can we combine for less indentation?